tendermint/src/rpc/endpoint/abci_query.rs

```diff
@@ -10,6 +10,7 @@ use serde::{Deserialize, Serialize};
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
 pub struct Request {
     /// Path to the data
+    #[serde(skip_serializing_if = "Option::is_none")]
     path: Option<Path>,
```

Comment:
Surprising that this can be empty. But yeah, in JSON everything seems possible :-D

Reply:
It's the Tendermint Go code that's surprising in most of these cases. Some of their definitions don't seem to make sense to me, but if we want to be compatible with a full node, we have to follow them.